core/capabilities/webapi/target/capability.go

```diff
@@ -0,0 +1,172 @@
+package target
```

Comment:
Does it make sense to put this in webapi instead and have the constructor be NewTarget? I think this will stutter less and also potentially lead to less conflicts: the unmodified name of the capability atm is target.NewCapability which doesn't say much

Reply:
I'm open to doing that. I think that would make the code more clear overall. We should enforce the same convention in other capabilities though to be consistent. (i.e. avoid `NewCapability` and use `New{$CapabilityName}Capability`)